catalog-info.yaml

```
@@ -0,0 +1,11 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'React-unit-tests-utils'
```

```suggestion
  name: 'react-unit-test-utils'
```

The name should match the repo name exactly.